packages/@blockprotocol/graph/src/stdlib/subgraph/builder.ts

```diff
@@ -0,0 +1,68 @@
+import { addEntitiesToSubgraphByMutation } from "../../internal/mutate-subgraph.js";
+import {
+  Entity,
+  EntityEditionId,
+  GraphResolveDepths,
+  Subgraph,
+} from "../../types.js";
+
+/**
+ * Builds a Subgraph from a given set of entities, which may include 'link entities' –
+ * i.e. entities that represent relationships between other entities – or other entities.
```

Comment:
```suggestion
 * Builds a Subgraph from a given set of entities, which may include 'link entities' –
 * i.e. entities that represent relationships between other entities 
```

I think that just confuses the sentence

Reply:
I previously had it as you have suggested, but then thought it might be misintrepreted as being "it may include link entities [and that's all it may include]"

how about
```suggestion
 * Builds a Subgraph from a given set of entities, which may include either or both of 'link entities' 
 * – i.e. entities that represent relationships between other entities – or non-link entities.
```

Reply:
```suggestion
 * Builds a Subgraph from a given set of entities, some (or all) of which may be 'link entities' –
 * i.e. entities that represent relationships between other entities – or other entities.
```